state-chain/runtime/src/migrations/sol_api_environment.rs

```diff
@@ -0,0 +1,96 @@
+use frame_support::traits::OnRuntimeUpgrade;
```

Comment:
we dont need this migration to be implemented here at the top level of the runtime. We can move it into the pallet level migration.

Reply:
@syan095

Reply:
Got it. Moved it into the Environment pallet.